posttroll/backends/zmq/ns.py

```diff
@@ -65,18 +82,21 @@ class ZMQNameServer:
 
     def __init__(self):
         """Set up the nameserver."""
-        self.running = True
-        self.listener = None
+        self.running:bool = True
+        self.listener:SocketReceiver|None = None
```

Comment:
Here the typehints should have spaces added for clarity.

Reply:
addressed in [18b6c1a](https://github.com/pytroll/posttroll/pull/72/commits/18b6c1af9277d306a8640a8b674861b68dec915d)